auto_py_to_exe/web/css/general.css

```diff
@@ -32,6 +29,20 @@
     --border-radius: 4px;
 }
 
+[data-dark-theme] {
```

Comment:
Small request: could we use a class instead? And this could then be `body.dark-theme`